Applications/WebPortal/WebApp/library/source/pages/account_page/compliance_page/rule_row.tsx

```diff
@@ -0,0 +1,225 @@
+import { css, StyleSheet } from 'aphrodite/no-important';
+import * as Nexus from 'nexus';
+import * as React from 'react';
+import { Transition } from 'react-transition-group';
+import { DisplaySize } from '../../../display_size';
+import { DropDownButton, HLine } from '../../../components';
+import { RuleExecutionDropDown } from './rule_execution_drop_down';
+import { ParametersList } from './parameter_list';
+import { ComplianceRuleEntry } from 'nexus';
```

Comment:
Nexus is considered "third party" so importing from it should be done before local imports.